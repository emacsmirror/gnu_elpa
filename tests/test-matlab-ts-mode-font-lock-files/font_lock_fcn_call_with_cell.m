% See comment in https://github.com/acristoffers/tree-sitter-matlab/issues/53
% This shouldn't be a parse error.

myfcn({'str'});

function myfcn(in)
    disp(in)
end

