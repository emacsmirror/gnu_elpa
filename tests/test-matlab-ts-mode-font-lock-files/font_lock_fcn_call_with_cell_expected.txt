c ccc ccccccccc ccc

c ccc ccccccc cc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
c cccc ccccccccc cc c ccccc cccccc

dddddbbSsssSbbD

kkkkkkkk fffffbvvb
    BBBBbddb
kkk

